publiccode.yml

```
@@ -0,0 +1,551 @@
+platforms:
+  - web
+developmentStatus: stable
+softwareVersion: "0.29.1"
+releaseDate: "2024-10-02"
+logo: "https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/logo.svg"
+softwareType: "standalone/web"
+
+categories:
+  - collaboration
+  - communications
+  - contact-management
+  - content-management
+  - event-management
+  - online-community
+  - survey
+  - web-collaboration
+
+description:
+  en:
+    shortDescription: >
+       The participatory democracy framework.
+       A generator and multiple gems made with Ruby on Rails
+    longDescription: >
+       Decidim is a Free and Open-Source digital infrastructure for participatory democracy.
+       Institutions and organizations around the world use Decidim to easily articulate governance and participatory processes: citizen initiatives, participatory budgets, strategic planning or collaborative legislation in an open and democratic way.
+       The project was originally developed for the Barcelona City Government, and now is maintained by the Decidim Free Software Association and a vibrant community.
+    features:
+      - "Participatory process: Run participatory processes to foster debate, assessment, and decision-making on issues that matter to your community."
+      - "Assembly: Create a space for your organization committees, choosing the level of privacy, to manage membership, meetings, or activities."
+      - "Initiatives: Collaborate with others to create bottom-up petitions by collecting signatures to achieve a common goal."
+      - "Proposal: Collect proposals from participants, enriched with geolocation and attachments, merge similar ones, and set a workflow for official approval or rejection."
+      - "Voting: Prioritize proposals using various voting methods, such as setting minimum and maximum thresholds, weighted voting, or participatory budgeting."
+      - "Meeting: Organize face-to-face or online meetings, manage registrations, and publish the agenda."
+      - "Participatory  text: Develop any document collaboratively by gathering suggestions and tracking all contributors."
+      - "Debate: Engage your community in a deliberation topic, and follow the discussion with different visualizations of the arguments."
+      - "Result: Ensure accountability in your participatory processes by converting proposals into results and tracking their implementation with the level of detail you prefer."
+      - "Post: Don’t forget to blog about your activities and share your learnings."
+    documentation: "https://docs.decidim.org"
+    apiDocumentation: "https://nightly.decidim.org/api"
+    awards:
+      - Sharing & Reuse Awards by European Commission (2019)
+      - Innovation in Politics Awards (2021)
+      - OSOR Awards by European Commission (2023)
+      - Recognized as a Digital Public Good by the DPGA (2023)
+
+legal:
+  license: AGPL-3.0-or-later
+  mainCopyrightOwner: Associació de Software Lliure Decidim
```

Might be duplicate, but would be more clear. 
```suggestion
  mainCopyrightOwner: Associació de Software Lliure Decidim
  repoOwner: Associació de Software Lliure Decidim
```